python/HeaderService/hutils.py

```diff
@@ -66,6 +61,9 @@ def configure_logger(logger, logfile=None, level=logging.NOTSET, log_format=None
         FORMAT_DATE = '%Y-%m-%d %H:%M:%S'
     formatter = logging.Formatter(FORMAT, FORMAT_DATE)
 
+    # Need to set the root logging level
```

Comment:
Please delete or expand on this comment. It is clear you are setting the log level from the code, which argues for deletion. But if there is some subtlety you are trying to document, please explain more.

Reply:
Let me know if makes sense to expand. If too obvious I'll delete.
The reason for changing the level for the root logging level, is because setting the level for each of the handler won't be recognized unless the root level is set at the appropriate level. For example, if you set the root logger to INFO, and all handlers to DEBUG, you still won't receive DEBUG messages on handlers.

Reply:
Hmm...I think the key is whether you'll need the information yourself if you modify the code later. If so I'd say a few more words. Otherwise I'd remove the comment. Either way I think it'll be an improvement.

Reply:
Will augment the comment.